README.md

```diff
@@ -2,31 +2,102 @@
 
 ![master](https://github.com/bazel-ios/rules_ios/workflows/CI-master/badge.svg)
 
-These rules provide some macros and rules that make it easier to build iOS
-application with Bazel. The heavy lifting of compiling, and packaging is
-still done by the existing
- [`objc_library` rule](https://bazel.build/reference/be/objective-c#objc_library)
-in Bazel, and by the
-[`swift_library` rule](https://github.com/bazelbuild/rules_swift/blob/master/doc/rules.md#swift_library)
-available from [rules_swift](https://github.com/bazelbuild/rules_swift).
-
-The goal of rules_ios has always been to allow seamlessly transitioning a set of targets from being built
-in Xcode to being built in bazel, with minimal-to-no code changes.
-In that vein, it acts as glue between the underlying rules (rules_apple and rules_swift and the native objc rules),
-applying defaults that mirror what Xcode does. 
-Additionally, rules_ios comes built-in with extension points that act as a quasi-toolchain,
-allowing users to customize things such as generated modulemaps.
+`rules_ios` is community developed Bazel rules to do iOS development with Bazel
+end to end.
 
-Bazel version required by current rules is [here](https://github.com/bazel-ios/rules_ios/blob/master/.bazelversion)
+It seamlessly bazel builds iOS applications originally written under Xcode with
+minimal-to-no code changes. It often re-uses ideas and code from `rules_swift`
+and `rules_apple` and it isn't tied to untested or unused features. It
+generates Xcode projects that _just work_ and makes using Apple Silicion with Bazel a breeze.
 
-**Xcode 12** and above supported, to find the last `SHA` with support for older versions see the list of git tags.
+### iOS Applications
 
-## Reference documentation
+_it supports all the primitives like apps, extensions, and widgets_
 
-[Click here](https://github.com/bazel-ios/rules_ios/tree/master/docs)
-for the documentation.
+```python
+load("@build_bazel_rules_ios//rules:app.bzl", "ios_application")
+
+ios_application(
+    name = "iOS-App",
+    srcs = glob(["*.m", "*.swift"]),
+    bundle_id = "com.example.ios-app",
+    minimum_os_version = "12.0",
+    visibility = ["//visibility:public"],
+)
+```
+
+### Xcode project generation
+
+_xcode project generaation that's tested and works end to end with open source
+remote execution and caching._
 
-## Quick setup
+```python
+load("@build_bazel_rules_ios//rules:xcodeproj.bzl", "xcodeproj")
+
+xcodeproj(
+    name = "MyXcode",
+    srcs = glob(["*.m", "*.swift"]),
```

Comment:
IIRC `srcs` is not a valid arg in this rule?

Reply:
Good catch - fixed 🏁